pkgs/by-name/be/benchexec/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3
+, libseccomp
+, nixosTests
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "benchexec";
+  version = "3.18";
+
+  src = fetchFromGitHub {
+    owner = "sosy-lab";
+    repo = "benchexec";
+    rev = version;
+    hash = "sha256-7A8B6PCDHVAl3FavB6YXLLFgAFJXn45Pp+sJgdpp5do=";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+  ];
+
+  propagatedBuildInputs =
+    [
+      # CPU Energy Meter is not added here, because BenchExec should call the
+      # wrapper configured via `security.wrappers.cpu-energy-meter`
+      # in `programs.cpu-energy-meter`, which will have the required
+      # capabilities to access MSR.
+      # If we add `cpu-energy-meter` here, BenchExec will instead call an executable
+      # without `CAP_SYS_RAWIO` and fail.
+      #pkgs.cpu-energy-meter
+      libseccomp.lib
```

Comment:
Is this required both as a propagated build input and as an environment variable? Maybe it's possible to drop one of them?

Reply:
I'll test that.